jcl/src/java.base/share/classes/openj9/internal/criu/InternalCRIUSupport.java

```diff
@@ -34,6 +34,7 @@ public final class InternalCRIUSupport {
 
 	private static native boolean isCRIUSupportEnabledImpl();
 	private static native boolean isCheckpointAllowedImpl();
+	private static native boolean disableCRIUSecProviderImpl();
 	private static native long getCheckpointRestoreNanoTimeDeltaImpl();
 	private static native long getLastRestoreTimeImpl();
```

Comment:
There's no good reason I can think of why these shouldn't be listed alphabetically.

While you're here, I don't think there are any warnings to be suppressed here: please remove lines 28-30.

Reply:
Updated

Reply:
This group of five native methods is still not sorted.

Reply:
Updated

Reply:
Sorted looks like this:
```java
	private static native boolean enableCRIUSecProviderImpl();
	private static native long getCheckpointRestoreNanoTimeDeltaImpl();
	private static native long getLastRestoreTimeImpl();
	private static native boolean isCRIUSupportEnabledImpl();
	private static native boolean isCheckpointAllowedImpl();
```

Reply:
Updated